index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>html and css cards quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+  <style type="quiz.css" media="screen">
```

Comment:
The style tag is only used for putting actual CSS in the HTML itself, i.e. you'd write the actual CSS in between the opening and closing style tag.  You've imported an external CSS document with the link tag on line 7, so you don't need these tags.